src/app/exercise/exercise.page/exercise.page.ts

```diff
@@ -41,23 +41,30 @@ export class ExercisePage {
   }
 
   onAnswerClick(answerConfig: AnswerConfig<string>): void {
+    
     if (this.isQuestionCompleted) {
       this.state.playAnswer(answerConfig);
       return;
     }
     const answer: string | null = answerConfig.answer;
+    
     if (!answer) {
       throw new Error(`Clicked answer is ${answer}`)
     }
     const isRight: boolean = this.state.answer(answer);
-    if (isRight) {
-      this.rightAnswer = answer;
+
+    if (isRight || true) {
```

Comment:
That condition will always be true. I believe you meant to write something else...